### PR TITLE
Add Zettersten 2020 norm data on colors and shapes

### PR DESCRIPTION
**Pull request by AnnikaTjuka**
This PR solves #690 .

The study in Zettersten and Lupyan comprises seven experiments with color and shape stimuli. I added the color stimuli to one list and the shape stimuli to another.

The mapping was pretty straightforward. In case of the colors, we could discuss whether we want to add colors which are not mapped yet.

### CODE REVIEW

**Comment by AnnikaTjuka** [comment]
Not sure why the checks failed. When I use `concepticon test', I get the info `all integrity tests passed: OK'.

**Comment by Schweikhard** [comment]
In case you didn't notice, you somehow added a lot of " to conceptlists.tsv now.

The mapping looks good.

**Comment by AnnikaTjuka** [comment]
> In case you didn't notice, you somehow added a lot of " to conceptlists.tsv now.
> 

I noticed. I tested your suggestion on defining columns as text for another list and LibreOffice kept those specification for the next list I opened.

**Comment by Schweikhard** [comment]
> I noticed. I tested your suggestion on defining columns as text for another list and LibreOffice kept those specification for the next list I opened.

Strange, for me it never keeps that and it certainly never adds quotes when there aren't any in the field in question. Just tested with this file and it doesn't add any quotes.

File: concepticondata/conceptlists.tsv [hunk 1]
```diff
@@ -305,3 +305,5 @@ Smeltzer-1995-300	Smeltzer, Brad and Smeltzer, Susan	1995		300	basic, questionna
...
 Morse-1996-375	Morse, Nancy L.	1996		375	areal, questionnaire	Spanish, English	Sáliba	https://www.sil.org/resources/archives/69059	Morse1996		This list of 375 items was used as a questionnaire for the indigenous South American language Sáliba. The items to be elicited were given both in English and Spanish. The author marked some items with asterisks that were taken from Swadesh-Rowe lists, most likely the [Swadesh 1955 100-item](:ref:Swadesh-1955-100) list, although a few of these marked items did not match any Swadesh list.		
+Zettersten-2020-24a	Zettersten, Martin and Lupyan, Gary	2020		24	naming test	English	English	https://doi.org/10.1016/j.cognition.2019.104135	Zettersten2020		This list comprises the color stimuli used in experiment 1A, 1B, 2A, 2B in [Zettersten and Lupyan (2020)](:bib:Zettersten2020). The study conducted seven experiments in which participants learned categories composed of colors and shapes that were either easy or more difficult to name in English.
```
Comment (Kristina-Pianykh) [inline concepticondata/conceptlists.tsv]: Do you think this list can be also tagged with 'colors' just for the purpose of findability (I'm thinking of someone who might specifically look for color wordlists)?

Comment (AnnikaTjuka) [reply concepticondata/conceptlists.tsv]: Yes, that's a good idea!

**Comment by AnnikaTjuka** [comment]
I also added the `colors` tag to the Nash and Haynie lists.

File: concepticondata/conceptlists.tsv [hunk 2]
```diff
@@ -305,3 +305,5 @@ Smeltzer-1995-300	Smeltzer, Brad and Smeltzer, Susan	1995		300	basic, questionna
...
+Zettersten-2020-24a	Zettersten, Martin and Lupyan, Gary	2020		24	naming test, colors	English	English	https://doi.org/10.1016/j.cognition.2019.104135	Zettersten2020		This list comprises the color stimuli used in experiment 1A, 1B, 2A, 2B in [Zettersten and Lupyan (2020)](:bib:Zettersten2020). The study conducted seven experiments in which participants learned categories composed of colors and shapes that were either easy or more difficult to name in English.		
+Zettersten-2020-16b	Zettersten, Martin and Lupyan, Gary	2020		16	naming test	English	English	https://doi.org/10.1016/j.cognition.2019.104135	Zettersten2020		This list comprises the shape stimuli used in experiment 3A and 3B in [Zettersten and Lupyan (2020)](:bib:Zettersten2020). The study conducted seven experiments in which participants learned categories composed of colors and shapes that were either easy or more difficult to name in English. The association values come from [Vanderplas and Garvin (1959)](:bib:Vanderplas1959).
```
Comment (chrzyki) [inline concepticondata/conceptlists.tsv]: Missing the `a` and `b` suffix in the `LIST_SUFFIX` columns

Comment (AnnikaTjuka) [reply concepticondata/conceptlists.tsv]: Thanks! I added them accordingly.

**Pull request merged**